src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.cs

```diff
@@ -218,34 +222,29 @@ protected virtual void SetupNewTextView(IVsTextView textView)
             }
         }
 
-        private void ConditionallyCollapseOutliningRegions(IVsTextView textView, IWpfTextView wpfTextView, Microsoft.CodeAnalysis.Workspace workspace, bool isOpenMetadataAsSource)
+        private void ConditionallyCollapseOutliningRegions(IVsTextView textView, IWpfTextView wpfTextView, bool collapseAllImplementations)
         {
             var outliningManagerService = this.Package.ComponentModel.GetService<IOutliningManagerService>();
             var outliningManager = outliningManagerService.GetOutliningManager(wpfTextView);
             if (outliningManager == null)
                 return;
 
-            if (!workspace.Options.GetOption(FeatureOnOffOptions.Outlining, this.RoslynLanguageName))
-            {
-                outliningManager.Enabled = false;
```

Comment:
This is where the old "Enter outlining mode" would actually turn off outlining.

The diff for this file is probably best viewed with whitespace off.